src/server/debugcmd.cc

```diff
@@ -489,7 +489,7 @@ void DebugCmd::Reload(CmdArgList args) {
   bool save = true;
 
   for (size_t i = 1; i < args.size(); ++i) {
-    string_view opt = absl::AsciiStrToUpper(ArgS(args, i));
+    string opt = absl::AsciiStrToUpper(ArgS(args, i));
```

Comment:
Could you rewrite the whole Debug command with CmdArgParser.

Reply:
Will do, just not in this PR. Got other things to fix :(